readme.md

```diff
@@ -367,6 +368,61 @@ AVA comes with builtin support for ES2015 through [Babel 6](https://babeljs.io).
 
 AVA includes typings for TypeScript. You have to setup transpilation yourself. When you set `module` to `commonjs` in your `tsconfig.json` file, TypeScript will automatically find the type definitions for AVA. You should set `target` to `es2015` to use Promises and async functions.
 
+### Babel Configuration for Test Scripts
+
+If you want to customize the babel transpiler for test files, you can do so by adding a `"babel"` key to the `ava` section in your `package.json` file.
+
+```json
+{
+	"ava": {
+		 "babel": {
+			 "presets": [
+					"es2015",
+					"stage-0",
+					"react"
+			 ]
+		 }
+	},
+}
+```
+
+In addition to specifying a custom Babel config, you can also use the special `"inherit"` keyword. When you do this, AVA will allow tests to be transpiled using the configuration defined in your `.babelrc` file or in package.json/babel. This way, your test files will be transpiled using the same options as your source files, but you won't have to define the options twice.
+
+```json
+{
+	"babel": {
+		"presets": [
+			"es2015",
+			"stage-0",
+			"react"
+		]
+	},
+	"ava": {
+		"babel": "inherit",
+	},
+}
+```
+
+Note: When configuring Babel for tests manually, the espower and transform-runtime plugins will be
```

Comment:
If we don't include transform-runtime, users will have to `import 'babel-polyfill'` at the top of each test file. This has not changed in the pull request. It was like this before. This is only a documentation change.